src/main/java/spotify/utils/HttpUtil.java

```diff
@@ -9,8 +9,24 @@
 
 import java.io.IOException;
 
+/**
+ * This class contains utility functions for use in the Authorization Code Flow.
+ * <p>
+ * For more information see: <a href="https://developer.spotify.com/documentation/general/guides/authorization-guide/#authorization-code-flow">Authorization Code Flow</a>
```

Comment:
You don't need to mention this in the Util class

Reply:
Don't need to mention what? Any of it?

Reply:
Sorry, line 15.

Reply:
OK. Ty